core/templates/pages/exploration-player-page/services/content-translation-manager.service.ts

```diff
@@ -247,9 +247,12 @@ export class ContentTranslationManagerService {
       }
     }
 
+    // DOMParser().parseFromString() creates a HTML document from
+    // the HTML string and it's body contains our required element
+    // as a childnode.
     const element = new DOMParser().parseFromString(
       card.getInteractionHtml(), 'text/html'
-    ).body;
+    ).body.childNodes[0] as HTMLElement;
```

Comment:
I think it would to do
```suggestion
    ).body.firstElementChild;
    if (element === null) {
      throw Error(̈́'<some explanation>')
    }
```

and then do the `as HTMLElement` down below on line 257.

But it is not a blocker and can be implemented in a subsequent PR.